packages/base/src/types/colormap.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "colormap/colorScale.js" {
+    export interface ColorScalePoint {
+      index: number;
+      rgb: [number, number, number];
+    }
+  }
```

Comment:
```suggestion
declare module "colormap/colorScale.js" {}
```

It looks like this is all we need. I think we should either write an empty stub or fully define the exports like so:

```typescript
declare module "colormap/colorScale.js" {
	interface ColorScalePoint {
	  index: number;
      rgb: [number, number, number];
	}
	
	type ColorScales = {
	  [key: string]: ColorScalePoint[];
	};
	
	declare const _exports: ColorScales;
	export = _exports;
}
```

I noticed no difference in the type inference from the compiler, so I think we should go with the minimal approach of an empty module declaration. Does that sound OK?